MobileWallet/Screens/Home/TransactionHistory/TransactionsTableViewController.swift

```diff
@@ -76,10 +99,61 @@ class TransactionsTableViewController: UITableViewController {
         })
     }
 
+    @objc private func refreshTable() {
+        if let wallet = TariLib.shared.tariWallet {
```

Comment:
guard would look better i think

Reply:
Cool, updated a bunch of places where I was using an if to a guard